src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2362,17 +2362,34 @@ int Matcher::max_vector_size(const BasicType bt) {
 }
 
 int Matcher::min_vector_size(const BasicType bt) {
-  int max_size = max_vector_size(bt);
-  // Limit the min vector size to 8 bytes.
-  int size = 8 / type2aelembytes(bt);
-  if (bt == T_BYTE) {
-    // To support vector api shuffle/rearrange.
-    size = 4;
-  } else if (bt == T_BOOLEAN) {
-    // To support vector api load/store mask.
-    size = 2;
+  // Theoretically, the minimal vector length supported by AArch64
+  // ISA and Vector API species is 64-bit. However, 32-bit or 16-bit
+  // vector length is also allowed for special Vector API usages.
+  int size;
+  switch(bt) {
+    case T_BOOLEAN:
+      // It needs to load/store a vector mask with only 2 elements
```

Comment:
```suggestion
      // Load/store a vector mask with only 2 elements
```
Same with the other cases.

Reply:
Thanks so much for your comment. I will fix them soon.